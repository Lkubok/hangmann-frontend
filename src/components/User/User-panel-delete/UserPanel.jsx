import React, { Component } from "react";
import "./UserPanel.scss";

export default class UserPanel extends Component {
  render() {
    return <div>UserPanel</div>;
  }
}
