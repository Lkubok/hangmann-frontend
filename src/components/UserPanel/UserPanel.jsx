import React, { Component } from "react";
import { connect } from "react-redux";
// import { Redirect } from "react-router-dom";
import "./UserPanel.scss";

export class UserPanel extends Component {
  render() {
    // const { isLogged } = this.props;
    return (
      <>
        <div>USERPANEL</div> }
      </>
    );
  }
}

const mapStateToProps = state => ({
  isLogged: state.appParamsReducer.isLogged
});

const mapDispatchToProps = dispatch => ({});

export default connect(
  mapStateToProps,
  mapDispatchToProps
)(UserPanel);
