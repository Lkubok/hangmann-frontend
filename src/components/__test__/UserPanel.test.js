import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import { UserPanel } from "../UserPanel/UserPanel";
Enzyme.configure({ adapter: new Adapter() });

describe("UserPanel component", () => {
  beforeEach(() => {});
  it("Should match snapshot", () => {
    const wrapper = shallow(<UserPanel />);
    expect(wrapper).toMatchSnapshot();
  });
});
