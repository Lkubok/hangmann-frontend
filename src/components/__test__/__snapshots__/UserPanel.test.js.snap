// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserPanel component Should match snapshot 1`] = `
<Fragment>
  <div />
</Fragment>
`;
